### PR TITLE
Update configure_partition.py

### PR DESCRIPTION
**Pull request by bretttegartms**
0 is Falsey- need to test on identity, not equality.

### CODE REVIEW

File: lib/python/treadmill_aws/cli/admin/cell/configure_partition.py [hunk 1]
```diff
@@ -58,9 +58,9 @@ def configure_partition_cmd(autoscale_max,
             data = {}
 
         autoscale = data.get('autoscale', {})
-        if autoscale_max:
+        if autoscale_max is not False:
```
Comment (andreikeis) [inline lib/python/treadmill_aws/cli/admin/cell/configure_partition.py]: why?

Comment (bretttegartms) [reply lib/python/treadmill_aws/cli/admin/cell/configure_partition.py]: 0 is falsey- it's not possible to set autoscale_min or _max to '0'.

Comment (andreikeis) [reply lib/python/treadmill_aws/cli/admin/cell/configure_partition.py]: "is not None" then?

Comment (bretttegartms) [reply lib/python/treadmill_aws/cli/admin/cell/configure_partition.py]: You're right, fixed.

**Pull request merged**